server/client_test.go

```diff
@@ -1893,3 +1893,25 @@ func TestClientStalledDuration(t *testing.T) {
 		})
 	}
 }
+
+func TestClientIPv6Address(t *testing.T) {
+	opts := DefaultOptions()
+	opts.Host = "0.0.0.0"
+	s := RunServer(opts)
+	defer s.Shutdown()
+
+	nc, err := nats.Connect(fmt.Sprintf("nats://[::1]:%v", opts.Port))
+	// Travis may not accept IPv6, in that case, skip the test.
```

Comment:
As I suspected, this test is useless on Travis since it does not support IPv6. But at least it will run on devs machine.

Reply:
ok